Formula/fastnetmon.rb

```diff
@@ -0,0 +1,60 @@
+class Fastnetmon < Formula
+  desc "DDoS detection tool with sFlow, Netflow, IPFIX and port mirror support"
+  homepage "https://github.com/pavel-odintsov/fastnetmon/"
+  url "https://github.com/pavel-odintsov/fastnetmon/archive/78baf3be83bb56e03528a96fe6df37eb20876cd9.tar.gz"
```

Comment:
we might prefer a tagged commit in here.

Reply:
Looks like we either want to wait for the 1.2.2 release or use the [1.2.1 release](https://github.com/pavel-odintsov/fastnetmon/archive/refs/tags/v1.2.1.tar.gz)

Reply:
Thank you for your feedback!

That's kind of chicken and egg problem. To add software to Homebrew we did multiple upstream changes like:

- https://github.com/pavel-odintsov/fastnetmon/commit/b3c5d89f0a40bc170c084977207a481905aeb3e3
- https://github.com/pavel-odintsov/fastnetmon/commit/78baf3be83bb56e03528a96fe6df37eb20876cd9
- https://github.com/pavel-odintsov/fastnetmon/commit/14481a07685171de11a6be61895d83a3dc51d763
- https://github.com/pavel-odintsov/fastnetmon/commit/038b25b49bc56ff9ec404a3cd185740a5cc164a5 

So we're still in process of changing things and cutting release right now we lead to multiple patches on top of it which will overcomplicated Formula.

After getting initial feedback from customers we may receive more things which will need to be addresses and then we may cut release.

Our current plan is to release it from tag and then after we do proper release we will do revision bump and will switrch to tag.

Is it possible to do it that way?

Thank you!

Reply:
It is not, https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Done, we've released 1.2.2 as stable release and I switched source code archive to tag.